### ISSUE TITLE
Add version bounds to library stanza in sitepipe.cabal

### ISSUE THREAD

**Issue by bandali** [ChrisPenner/SitePipe#8]
It's considered good practice to specify lower and upper version bounds on the dependencies of the `library` stanza in `sitepipe.cabal`.

**Comment by ChrisPenner** [comment]
I've always put off putting boundaries in because I don't really know of a good way to know what the lower limit should be; is there an easy way to figure that out or do I just keep lowering it until something breaks?

I definitely do want to add these; just not sure how to come up with solid bounds. I suppose I can just put something in there and if people request lower bounds we can go from there?

**Comment by bandali** [comment]
Yeah I've wondered the same. I've bookmarked the [PVP Specification](https://pvp.haskell.org/) and [FAQ](https://pvp.haskell.org/faq/), but even those don't say much about lower bounds.

What I've noticed I've been doing is to first pick a lower bound for `base` (i.e. what's the lowest GHC version I'd like to support) then pick the lowest version of each dependency that doesn't break the build. Then basically proceed as you said and change the lower bounds when needed: e.g. people requesting decrease in lower bounds (if possible), or having to increase lower bounds to be able to support new GHC (what I've done in #11 with `pandoc` to be able to support GHC 8.2.1).
